### PR TITLE
Added test for IntTransform2D

### PR DESCRIPTION
**Pull request by marcusnaslund**
Based on great tests by @molhamkhalil this fixes the last part of #788 
Also changed `toString` to conform with the float version.

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Peer review @thomasfanell ?

File: test/geometry/IntTransform2DTest.ooc [hunk 1]
```diff
@@ -152,6 +152,155 @@ IntTransform2DTest: class extends Fixture {
...
+			expect(transform c, is equal to(0))
+			expect(transform d, is equal to(2))
+			expect(transform e, is equal to(1))
+			expect(transform f, is equal to(0))
+			expect(transform g, is equal to(-7))
+			expect(transform h, is equal to(3))
+			expect(transform i, is equal to(1))
+		})
+		this add("skewX", func {
+			transform := this transform0 skewX(Float pi / 6)
```
Comment (thomasfanell) [inline test/geometry/IntTransform2DTest.ooc]: `6.0f`? Same on line 238 and 286

File: test/geometry/IntTransform2DTest.ooc [hunk 2]
```diff
@@ -152,6 +152,155 @@ IntTransform2DTest: class extends Fixture {
...
+			expect(reflectY c, is equal to(0))
+			expect(reflectY d, is equal to(0))
+			expect(reflectY e, is equal to(-1))
+			expect(reflectY f, is equal to(0))
+			expect(reflectY g, is equal to(0))
+			expect(reflectY h, is equal to(0))
+			expect(reflectY i, is equal to(1))
+		})
+		this add("setTranslation", func {
+			transform := this transform0 setTranslation(IntVector2D new(-7.0f, 3.0f))
```
Comment (thomasfanell) [inline test/geometry/IntTransform2DTest.ooc]: `IntVector2D` take floats?

**Comment by thomasfanell** [comment]
Other than that, I'm a happy camper.

**Comment by marcusnaslund** [comment]
Ah thanks, this is what happens when it's too easy to port a Float-based test to Int.

**Comment by marcusnaslund** [comment]
I'm putting this to final review in your absence. May the gods forgive me if you had other comments.

**Pull request merged**